examples/spin-timer/deps/spin/variables.wit

```diff
@@ -0,0 +1,13 @@
+package fermyon:spin@2.0.0
```

Comment:
All of these comments obviously mostly apply to the wit file in "wit/preview2" (which I originally thought this file was).

Reply:
Yep (sorry it's confusing this WIT exists in an example because pre-registry days). But nonetheless they should be identical barring the package statement.